starter-code/server.js

```diff
@@ -34,15 +35,15 @@ app.use(express.static('./public'));
 // REVIEW: Routes for requesting HTML resources
 app.get('/new', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // #5; Article.prototype.toHtml; Update;
```

Comment:
this function is not interacting with `Article.prototype.toHtml`
CRUD method update does not apply